install_python_ubuntu.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```

Then you can get rid of '&&' operators